source/firmware_update_test/firmware_update_test.cpp

```diff
@@ -31,19 +31,19 @@
 
 void copyAppToSDCard(uint32_t firmware_size)
 {
-    tr_info("Copy active firmware to slot 0\r\n");
+    tr_info("Copy active firmware to slot 0");
 
     arm_uc_firmware_details_t details = { 0 };
 
-    tr_info("calculate firmware SHA256\r\n");
+    tr_info("calculate firmware SHA256");
```

Comment:
Please remove these debug messages which are basically just telling what is already in our design document.